content/influxdb/v2.0/reference/cli/influx/v1/dbrp/list.md

```diff
@@ -0,0 +1,37 @@
+---
+title: influx v1 dbrp list
+description: >
+  The `influx v1 dbrp list` command lists and searches DBRP mappings in the InfluxDB 1.x compatibility API.
+menu:
+  influxdb_2_0_ref:
+    name: influx v1 dbrp list
+    parent: influx v1 dbrp
+weight: 101
+influxdb/v2.0/tags: [dbrp]
+---
+
+The `influx v1 dbrp list` command lists and searches DBRP mappings in the [InfluxDB 1.x compatibility API](/influxdb/v2.0/reference/api/influxdb-1x/).
+
+## Usage
+```
+influx v1 dbrp list [flags]
+```
+
+## Flags
+
+| Flag |                   | Description                                                                                | Input type | {{< cli/mapped >}}      |
+|------|-------------------|--------------------------------------------------------------------------------------------|------------|-------------------------|
+| `-c` | `--active-config` | Config name to use for command                                                             | string     | `$INFLUX_ACTIVE_CONFIG` |
+| `-b` | `--bucket-id`     | Bucket ID                                                                                  |            |                         |
+|      | `--configs-path`  | Path to the influx CLI configurations (default: `~/.influxdbv2/configs`)                   | string     | `$INFLUX_CONFIGS_PATH`  |
+| `-d` | `--db`            | InfluxDB v1 database to map from                                                                    |            |                         |
+|      | `--default`       | Specify if this mapping represents the default retention policy for the database specified |            |                         |
```

Comment:
not sure the following apply here:
`--default`  -- doesn't make sense in the context of `list`.

Reply:
@timhallinflux It's in the help text. I think @sanderson is planning on filing an issue about this.

Reply:
ok!